azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt

```diff
@@ -122,16 +120,23 @@ internal class ParticipantListView(
     }
 
     private fun generateBottomCellItems(
```

Comment:
do we need additional list?

Please try with:
```private fun generateBottomCellItems(
        remoteParticipantCellModels: List<ParticipantListCellModel>,
    ): List<BottomCellItem> {
        val bottomCellItems = mutableListOf<BottomCellItem>()
        // since we can not get resources from model class, we create the local participant list cell
        // with suffix in this way
        val localParticipant = viewModel.createLocalParticipantListCell(resources.getString(R.string.azure_communication_ui_call_local_participant_suffix))

        bottomCellItems
            .add(
                generateBottomCellItem(localParticipant.displayName, localParticipant.isMuted)
            )

        for (remoteParticipant in remoteParticipantCellModels) {
            bottomCellItems.add(
                generateBottomCellItem(
                    if(remoteParticipant.displayName.isEmpty()) "Unnamed Participant" // create xml string
                    else remoteParticipant.displayName,
                    remoteParticipant.isMuted)
            )
        }

        return bottomCellItems.sortedWith(compareBy(String.CASE_INSENSITIVE_ORDER, { it.title!! }))
    }```

Reply:
addressed